packages/styled-docs/pages/colors.mdx

```diff
@@ -0,0 +1,97 @@
+import {
+  ColorPalette,
+  ColorPalettes,
+  ColorWrapper,
+} from "../components/ColorPalette";
+import ThemeParser from "../components/ThemeParser";
+
+# Colors
+
+Add a `theme.colors` object to provide colors for your project. By default these
+colors can be referenced by the `color`, `borderColor`, and `backgroundColor`,
+`fill`, `stroke`, styles.
+
+We recommend adding palette that go from `10` - `100`.
+
+<ThemeParser theme="colors" />
+
+### Orange
+
+<ColorWrapper>
+  <ColorPalettes color="orange" />
+</ColorWrapper>
+
+### Yellow
+
+<ColorWrapper>
+  <ColorPalettes color="yellow" />
+</ColorWrapper>
+
+### Red
+
+<ColorWrapper>
+  <ColorPalettes color="red" />
+</ColorWrapper>
+
+### Magenta
+
+<ColorWrapper>
+  <ColorPalettes color="magenta" />
+</ColorWrapper>
+
+### Purple
+
+<ColorWrapper>
+  <ColorPalettes color="purple" />
+</ColorWrapper>
+
+### Blue
+
+<ColorWrapper>
+  <ColorPalettes color="blue" />
+</ColorWrapper>
+
+### Green
+
+<ColorWrapper>
+  <ColorPalettes color="green" />
+</ColorWrapper>
+
+### Teal
+
+<ColorWrapper>
+  <ColorPalettes color="teal" />
+</ColorWrapper>
+
+### Cyan
+
+<ColorWrapper>
+  <ColorPalettes color="cyan" />
+</ColorWrapper>
+
+### Gray
+
+<ColorWrapper>
+  <ColorPalettes color="gray" />
+</ColorWrapper>
+
+### Black Transparent
```

Comment:
Use Sentence case for H3, H4, H5, H6

Reply:
How about using `### Black text` ?

Reply:
> How about using `### Black text` ?

But it's talk about color, not specific for text. I'll think long and hard.

Reply:
Simply “Black” is fine if we don’t have a better naming at the moment.

Reply:
Good suggestion, thanks!